README.md

```diff
@@ -1,9 +1,15 @@
 # Contra
-> The Contra utility converts the so called "C Style Markup Language" to HTML/XML and back.
+> The Contra utility converts [C Style Markup Language](definition of C style markup) to HTML/XML and back.
```

Comment:
Is the C Style markup language something that you created?  Is there a formal project page for it?  Link to that here.  Or if it's your own creation, link to the docs for it.